tools/linter_lib/custom_check.py

```diff
@@ -489,16 +505,24 @@ def build_custom_checkers(by_lang):
                          "static/templates/settings/bot-settings.handlebars",
                          "templates/zerver/email_log.html"])},
         {'pattern': "placeholder='[^{]",
-         'description': "`placeholder` value should be translatable."},
+         'description': "`placeholder` value should be translatable.",
+         'good_lines': ['<input class="stream-list-filter" type="text" placeholder="{{ _(\'Search streams\') }}" />'],
+         'bad_lines': ["<input placeholder='foo'></input>"]},
         {'pattern': "aria-label='[^{]",
-         'description': "`aria-label` value should be translatable."},
+         'description': "`aria-label` value should be translatable.",
+         'good_lines': ['<button class="button small btn-danger deactivate btn-danger" aria-label="{{ \'Deactivate stream\' }}">'],
+         'bad_lines': ["<button aria-label='foo'></button>"]},
         {'pattern': 'aria-label="[^{]',
          'description': "`aria-label` value should be translatable.",
          'exclude': set(["docs/_templates/layout.html"])},
         {'pattern': 'script src="http',
-         'description': "Don't directly load dependencies from CDNs.  See docs/subsystems/front-end-build-process.md"},
+         'description': "Don't directly load dependencies from CDNs.  See docs/subsystems/front-end-build-process.md",
+         'good_lines': ['<script src="{{ render_bundle("translations") }}"></script>'],
```

Comment:
You should do a search for `render_bundle` to see how we use it (eg `git grep render_bundle`)
There is no need for the script tag.